cdap-proto/src/main/java/io/cdap/cdap/proto/ProgramRunStatus.java

```diff
@@ -33,6 +33,7 @@
   RUNNING,
   SUSPENDED,
   RESUMING,
+  STOPPING,
```

Comment:
how does the program transitions look like if we are adding new state in `canTransitionTo ` method?

Reply:
As discussed offline, these changes will be made in one of the following PRs

Reply:
can you file a jira and add a TODO here for tracking?

Reply:
missed?